R/RSocrata.R

```diff
@@ -355,15 +397,18 @@ read.socrata <- function(url, app_token = NULL, email = NULL, password = NULL,
 #' # Check schema definition for metadata
 #' attributes(df)
 #' @importFrom jsonlite fromJSON
-#' @importFrom httr parse_url
+#' @importFrom httr GET build_url parse_url content user_agent
 #' @export
 ls.socrata <- function(url) {
   url <- as.character(url)
   parsedUrl <- httr::parse_url(url)
   if(is.null(parsedUrl$scheme) | is.null(parsedUrl$hostname))
     stop(url, " does not appear to be a valid URL.")
   parsedUrl$path <- "data.json"
-  data_dot_json <- jsonlite::fromJSON(httr::build_url(parsedUrl))
+  #Download data
+  response <- httr::GET(httr::build_url(parsedUrl), user_agent(fetch_user_agent()))
```

Comment:
no `httr::` in front of `user_agent`
(but this isn't an error)

Reply:
Agree, I'll make the change.